explainer.md

```diff
@@ -532,26 +533,40 @@ interface VRSession : EventTarget {
   attribute EventHandler onfocus;
   attribute EventHandler onresetpose;
 
-  VRSourceProperties getSourceProperties(optional double scale);
+  readonly attribute VRDevicePose pose;
 
   Promise<VRFrameOfReference> createFrameOfReference(VRFrameOfReferenceType type);
-  VRDevicePose? getDevicePose(VRCoordinateSystem coordinateSystem);
 
+  Promise<VRPresentationFrame> requestVRPresentationFrame();
+  
   Promise<void> endSession();
 };
 
 //
-// Pose
+// Presentation Frame, Device Pose, and Views
 //
+interface VRPresentationFrame {
+  readonly attribute FrozenArray<VRView> views;
 
-interface VRDevicePose {
-  readonly attribute Float32Array leftProjectionMatrix;
-  readonly attribute Float32Array leftViewMatrix;
+  VRDevicePose? getDevicePose(VRCoordinateSystem coordinateSystem);
+
+  VRWebGLViewport getViewport(VRWebGLLayer layer, VRView? view);
```

Comment:
I think I prefer the overloaded version to introducing more object hierarchy. For the sake of this PR I'd say just go with it as is, and we can have a more targeted discussion in a separate issue if needed.